apps/are_form/views/accessibility.html

```diff
@@ -15,28 +15,28 @@
         <li>{{#t}}accessibility.intro-p2-li4{{/t}}</li>
         <li>{{#t}}accessibility.intro-p2-li5{{/t}}</li>
     </ul>
-    <p class="govuk-body">{{#t}}accessibility.intro-p3-pre-link{{/t}} <a href="https://mcmw.abilitynet.org.uk/">{{#t}}accessibility.intro-p3-link{{/t}}</a> {{#t}}accessibility.intro-p3-post-link{{/t}}</p>
-    <p class="govuk-body govuk-!-font-weight-bold">Reporting accessibility problems with this service</p>
+    <p class="govuk-body">We’ve also made the text as simple as possible to understand. <a href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+    <h3 class="govuk-heading-m">Reporting accessibility problems with this service</h3>
     <p class="govuk-body">We’re always looking to improve the accessibility of our services. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, email: <a href="mailto:hof-accessibility@digital.homeoffice.gov.uk">hof-accessibility@digital.homeoffice.gov.uk</a>.</p>
-    <p class="govuk-body govuk-!-font-weight-bold">Enforcement procedure</p>
+    <h3 class="govuk-heading-m">Enforcement procedure</h3>
     <p class="govuk-body">The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, 
       <a href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS)</a>.
       If you are in Northern Ireland and are not happy with how we respond to your complaint you can contact the
-      <a href="https://www.equalityni.org/Home">the Equalities Commission for Northern Ireland</a> who are responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’) in Northern Ireland.
+      <a href="https://www.equalityni.org/Home"> Equalities Commission for Northern Ireland</a> who are responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’) in Northern Ireland.
     </p>
```

Comment:
move this to end of line 25 (behind "Northern Ireland"?   Super minor of course - just a thought.